src/pages/api/auth/[...nextauth].ts

```diff
@@ -103,16 +103,13 @@ export default NextAuth({
 					token.access_token = access_token
 					token.refresh_token = refresh_token
 				} catch (err) {
-					let errorType = AuthErrors.RefreshAccessTokenError
-
-					if (err.error === 'token_inactive') {
-						errorType = AuthErrors.RefreshAccessTokenExpiredError
-					}
```

Comment:
Why don't we care about the error type here anymore? 💭

Reply:
Well we're not doing anything specific with it anymore. Were just signing out on either error type at the moment.

Reply:
ah yes you beat me to it 😄